apps/nextjs-website/src/messages/it.json

```diff
@@ -123,13 +123,18 @@
             "description": "",
             "noWebinars": "Stiamo preparando i prossimi eventi, controlla più avanti.",
             "title": {
-                "dontLoseNext": "Non perderti il prossimo webinar",
+                "dontLoseNext": "Non perderti {webinars, plural, =1 {il prossimo} other {i prossimi}} webinar",
```

Comment:
What about `0` value? Does the title goes to plural?
Is it possible to use a boolean instead of a number?